.kuzzlerc

```diff
@@ -7,6 +7,12 @@
   */
   "serviceSettingsCollection": "services",
 
+  "default": {
```

Comment:
I'm not fond of adding sublevels here, unless we start grouping services configuration together: after reading the file several times, I think this added structure won't make much sense to newcomers.

What do you think of `serviceSettingsTimeout`, at the same level than `serviceSettingsCollection`?
Not fan of it, but it would be homogeneous.

We may have to better structure this file  sooner or later.